bcdc_apitests/tests/resources/test_resources.py

```diff
@@ -11,10 +11,10 @@
 logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
 
-# looks like a resourc has to be part of a package, ie the package_id needs to
+# looks like a resource has to be part of a package, ie the package_id needs to
 # be populated.  Call the fixtures
 def test_resource_create(ckan_url, ckan_rest_dir, ckan_auth_header,
-                         resource_data, package_create_if_not_exists):
+                         resource_data,package_create_if_not_exists):
```

Comment:
thinking we should get you set up with a linter.  Generally I'm trying to follow the style guildelines that pylint checks for... http://pylint-messages.wikidot.com/messages:c0326